### PR TITLE
Setup & Controllers

### PR DESCRIPTION
**Pull request by brytebee**
### Setup & Controllers 🎮 
In this PR, I have done the following:
- Installed Rails using `rails new`.
- Installed PostgreSQL database.
- Installed `rubocop `and `stylelint` linter files.
- Added routes.
- Added controllers.
- Added views.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def posts; end
+
+  def show; end
+
+  def new; end
```
Comment (Meltrust) [inline app/controllers/posts_controller.rb]: - [ ] The "posts" and "new" actions are not needed. Please, kindly remove them.

Only these are needed:

```ruby
  def index; end

  def show; end
```

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def users; end
+
+  def show; end
+
+  def new; end
```
Comment (Meltrust) [inline app/controllers/users_controller.rb]: - [ ] The "new" and the "users" actions are not needed.  Kindly remove them.  You only need:


```ruby
  def index; end

  def show; end
```

File: app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>Create a new user <button>New user</button></p>
```
Comment (Meltrust) [inline app/views/users/new.html.erb]: - [ ] This file is not needed.  We only need "index" and "show" files.  Kindly delete this file.

File: app/views/posts/new.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<p>Create a new post <button>New user</button></p>
```
Comment (Meltrust) [inline app/views/posts/new.html.erb]: - [ ] This file is not needed.  We only need "index" and "show" files.  Kindly delete this file.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @brytebee,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice routes config!✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @brytebee 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You asked for the changes in a polite and friendly way.
- Github inline comments feature is used properly.
- You documented your work professionally.✔

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**